tests/examples/NamespacePrivateLinter/namespace_private_in_no_ns.php.expect

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
if they're short enough, tests with no errors should generally be tuples in the `getCleanExamples()` part of the test